backend/src/main/scala/cromwell/backend/backend.scala

```diff
@@ -26,7 +26,7 @@ import scala.util.Try
 /**
   * For uniquely identifying a job which has been or will be sent to the backend.
   */
-case class BackendJobDescriptorKey(call: CommandCallNode, index: Option[Int], attempt: Int) extends CallKey {
+case class BackendJobDescriptorKey(call: CommandCallNode, index: Option[Int], attempt: Int, memoryDoubleMultiplier: Int = 1) extends CallKey {
```

Comment:
Perhaps call the field `memoryMultiplier`? While here, I'm also a fan of `Double` multipliers vs. `Int`.